framework/console/controllers/MessageController.php

```diff
@@ -477,15 +477,15 @@ protected function saveMessagesToDb($messages, $db, $sourceMessageTable, $messag
         }
         
         foreach ($currentLanguages as $currentLanguage) {
-          if (!in_array($currentLanguage, $languages)) {
-            $deleted=$db->createCommand()->delete($messageTable, "language=:language", [
+            if (!in_array($currentLanguage, $languages)) {
+                $deleted=$db->createCommand()->delete($messageTable, 'language=:language', [
```

Comment:
No spaces around `=`.

Reply:
This will be fixed by PSR12, I did not push it yet.